### PR TITLE
Added VertX resource

### PR DESCRIPTION
**Pull request by huntley**
Review not necessary yet. Waiting for Vert.x release version and updated pom.xml example to make final update.

### CODE REVIEW

**Comment by tradej** [comment]
add to whitelist

File: docs/topics/ref_additional-vertx-resources.adoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 * link:https://www.reactivemanifesto.org/[The Reactive Manifesto]
 * link:http://vertx.io[{VertX} project]
+* link:https://www.manning.com/books/vertx-in-action[{VertX} in Action]
```
Comment (Ladicek) [inline docs/topics/ref_additional-vertx-resources.adoc]: I think using `{VertX}` here is wrong, as that (IIRC) expands to "Eclipse Vert.x", but the book is really only called Vert.x in Action: https://www.manning.com/books/vertx-in-action

Comment (huntley) [reply docs/topics/ref_additional-vertx-resources.adoc]: I thought that odd myself but I followed the convention used for all of the links. I will verify that the other links to make sure  they are correct as well.

File: docs/topics/ref_additional-vertx-resources.adoc [hunk 2]
```diff
@@ -3,10 +3,11 @@
 
 * link:https://www.reactivemanifesto.org/[The Reactive Manifesto]
 * link:http://vertx.io[{VertX} project]
+* link:https://www.manning.com/books/vertx-in-action[Vert.x in Action]
 * link:http://middlewareblog.redhat.com/2017/05/04/vert-x-for-reactive-programming-in-red-hat-openshift-application-runtimes/[{VertX} for Reactive Programming in Red Hat OpenShift Application Runtimes]
 * link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java]
 * link:https://developers.redhat.com/promotions/vertx-cheatsheet/[{VertX} Cheat Sheet for Developers]
-* link:http://escoffier.me/vertx-hol/#_vert_x[{VertX} - From zero to (micro)-hero]
+* link:http://escoffier.me/vertx-hol/#_vert_x[{Vert.x - From zero to (micro)-hero]
```
Comment (tradej) [inline docs/topics/ref_additional-vertx-resources.adoc]: There's an extra opening bracket before _Vert.x._

**Comment by huntley** [comment]
Add GA version. I think the runtime guide is ready. Please let me know if you have any feedback. Thanks!

**Comment by huntley** [comment]
Fixed Vert.x version.

File: docs/topics/ref_additional-vertx-resources.adoc [hunk 3]
```diff
@@ -3,10 +3,11 @@
 
 * link:https://www.reactivemanifesto.org/[The Reactive Manifesto]
 * link:http://vertx.io[{VertX} project]
+* link:https://www.manning.com/books/vertx-in-action[Vert.x in Action]
 * link:http://middlewareblog.redhat.com/2017/05/04/vert-x-for-reactive-programming-in-red-hat-openshift-application-runtimes/[{VertX} for Reactive Programming in Red Hat OpenShift Application Runtimes]
 * link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java]
 * link:https://developers.redhat.com/promotions/vertx-cheatsheet/[{VertX} Cheat Sheet for Developers]
-* link:http://escoffier.me/vertx-hol/#_vert_x[{VertX} - From zero to (micro)-hero]
+* link:http://escoffier.me/vertx-hol/#_vert_x[Vert.x - From zero to (micro)-hero]
```
Comment (cescoffier) [inline docs/topics/ref_additional-vertx-resources.adoc]: The kubernetes one my be better: http://escoffier.me/vertx-kubernetes/. I let you decide.

**Comment by tradej** [comment]
Merging so that we have as much work done. @huntley, if you want to add the resource Clement mentions, please file a new PR.

**Pull request merged**